api/pom.xml

```diff
@@ -170,6 +170,10 @@
         </dependency>
 
         <!-- End OpenMRS modules -->
+        <dependency>
```

Comment:
Do we need this dependency?

Reply:
yes , the growth chart API heavily depends on it